src/app/maps/m6/directives/accordion.directive.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { AccordionDirective } from './accordion.directive';
+
+describe('AccordionDirective', () => {
+  it('should create an instance', () => {
+    const directive = new AccordionDirective();

```

Comment:
this test does not even compile

Reply:
correction - none of the tests compile.

Reply:
we use it before, but now it's unused, i remove it